how-we-work/all-hands-meeting.html.md

```diff
@@ -19,33 +44,17 @@ All Hands events need some work from the meeting organizer both to prepare the m
 to be sure everyone is aware of the meeting and what to expect from it. If you are in charge of
 organizing the All Hands, simply follow these tips to make it a success!
 
-### One day before the meeting
-
+- **Check the calendar event.** We have a shared Nebulab calendar that everyone has joined during
```

Comment:
"that everyone has joined" -> "that everyone joins", since it's a company policy that is still in place.